vagrant/boxes.d/99-local.yaml.example

```diff
@@ -1,7 +1,23 @@
 ---
-centos8-katello-devel:
+centos9-katello-devel:
   primary: true
-  box: centos8-stream
+  box: centos9-stream
+  ansible:
+    playbook: 'playbooks/katello_devel.yml'
+    group: 'devel'
+    variables:
+      ssh_forward_agent: true
+      foreman_devel_github_push_ssh: True
+      katello_devel_github_username: <REPLACE ME>
+      foreman_repositories_environment: staging
+      katello_repositories_environment: staging
+      foreman_installer_options:
+        - "--foreman-proxy-content-enable-ostree=true"
+        - "--katello-devel-modulestream-nodejs=18"
```

Comment:
Musing: we _could_ remove this with a merge of https://github.com/theforeman/puppet-katello_devel/pull/300 , which (among other fixes) makes NodeJS 18 the default, but I lean towards keeping it here anyway for the purpose of discoverability.

Reply:
Perhaps comment it out as an example?